src/open_options.rs

```diff
@@ -71,9 +74,42 @@ impl OpenOptions {
             truncate: false,
             create: false,
             create_new: false,
+            buffer_size: 0,
+            replication: 0,
+            blocksize: 0,
         }
     }
 
+    /// Sets size of buffer for read/write.
+    ///
+    /// Pass `0` if you want to use the default configured values.
+    ///
+    /// `0` by default.
+    pub fn with_buffer_size(&mut self, buffer_size: c_int) -> &mut Self {
+        self.buffer_size = buffer_size;
+        self
+    }
+
+    /// Sets block replication.
+    ///
+    /// Pass `0` if you want to use the default configured values.
+    ///
+    /// `0` by default.
+    pub fn with_replication(&mut self, replication: c_short) -> &mut Self {
```

Comment:
The same.

Reply:
`c_short` is currently uniquely points to `i16` for all platforms. But in case there would come with a new platform in the future and it would refer to another type otherwise, I think we should just stick to `c_short` for the maximum compatibility.

Reply:
As mentioned in https://github.com/Xuanwo/hdrs/pull/136#discussion_r1839714010, using `ffi` in the public API of a high-level crate like `hdrs` doesn't sit well with me.